### PR TITLE
Fix reblog confirm

### PR DESCRIPTION
**Pull request by charlag**
These are some nasty bugs. I'm sorry I didn't re-test it properly.

Idk if you want to release everything in develop with the next release, I could send this PR against master too.

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java [hunk 1]
```diff
@@ -616,23 +616,19 @@ protected void setupButtons(final StatusActionListener listener, final String ac
...
-                    listener.onReblog(!buttonState, position);
-                }
-                if (statusDisplayOptions.confirmReblogs()) {
-                    showConfirmReblogDialog(listener, statusContent, buttonState, position);
-                    return false;
+                    if (statusDisplayOptions.confirmReblogs()) {
+                        showConfirmReblogDialog(listener, statusContent, buttonState, position);
+                        return false;
+                    } else {
+                        listener.onReply(position);
```
Comment (Tak) [inline app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java]:
```suggestion
                        listener.onReblog(!buttonState, position);
```

File: app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java [hunk 2]
```diff
@@ -616,23 +616,19 @@ protected void setupButtons(final StatusActionListener listener, final String ac
                                 StatusDisplayOptions statusDisplayOptions) {
 
         avatar.setOnClickListener(v -> listener.onViewAccount(accountId));
-        replyButton.setOnClickListener(v -> {
-            int position = getAdapterPosition();
-            if (position != RecyclerView.NO_POSITION) {
-                listener.onReply(position);
-            }
-        });
```
Comment (Tak) [inline app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java]: Why remove this block?

**Comment by Tak** [comment]
For what it's worth,  I've been running this branch (with the two changes I suggested) for a few weeks now with no issues

**Comment by charlag** [comment]
Oh, I'm sorry, I forgot about the PR, it looked like I ignored @Tak which I didn't intend at all.
I don't have a lot of energy to finish this but I'll try to do it soon.

**Comment by Tak** [comment]
No worries 🙂

**Comment by charlag** [comment]
Sorry for this mess of a PR, I fixed mentioned issues, rebased it and we should squash it before merging

**Pull request merged**